### ISSUE TITLE
Dynamic dialogue box

### ISSUE THREAD

**Issue by cheny36** [allenjiang17/theSearchForEden#16]
Component object that produces text one letter at a time, similar to dialogue boxes in pokemon.

**Comment by cheny36** [comment]
TODO:

- on click, finishes typing out the text
- option to split text if too large
- predict the size of the text box so that things don't move around as its typing

**Comment by allenjiang17** [comment]
should we also get the event box to do the same? seems weird that the location description is doing that but the event description is not. but then you'd need it to be sequential.

i think maybe we don't do it for the location description (since that's static), but we do it for the event description since that's the more interesting/dynamic part

**Comment by Larinitus** [comment]
Agreed with Allen. I think it's more important to have the event description displayed one text letter at a time.
